astropy/cosmology/tests/test_flrw.py

```diff
@@ -833,6 +839,15 @@ class TestFlatSomeFLRW(FlatFLRWMixinTest, TestSomeFLRW):
             ...
     """
 
+    def setup_class(self):
+        """Setup for testing.
+
+        This passes setup up/along the inheritance chain, then removes
```

Comment:
Comments are most useful if they explain *why* something is done, not just *what* is done. I'd write here something along the lines of "Set up as for regular FLRW test class, but remove dark energy component since for flat cosmologies that is calculated automatically using the flatness constraint."

Reply:
SGTM. I borrowed from your suggestion for the first bit, but the removal is actually because flat cosmologies are forbidden from having Ode0 by `__init_subclass__` (to ensure it's calculated).